hw/ip/edn/dv/edn_edn0_sim_cfg.hjson

```diff
@@ -0,0 +1,13 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{
+  // Name of the sim cfg variant
+  variant: edn0
+
+  // Import the base entropy_src sim_cfg file
+  import_cfgs: ["{proj_root}/hw/ip/edn/dv/edn_base_sim_cfg.hjson"]
+
+  build_opts: ["+define+NUM_END_POINTS=8"]
```

Comment:
Does the `+define+` syntax work across different simulators?

Reply:
Yes it does, we use this also for blocks simulated with Xcelium (e.g. ENTROPY_SRC, AES etc.).